lightblue/src/main/java/org/esbtools/eventhandler/lightblue/FindRequests.java

```diff
@@ -25,10 +25,23 @@
 import com.redhat.lightblue.client.request.data.DataFindRequest;
 
 import org.esbtools.eventhandler.lightblue.model.DocumentEventEntity;
+import org.esbtools.lightbluenotificationhook.NotificationEntity;
 
 public abstract class FindRequests {
-    public static DataFindRequest newNotificationsForEntitiesUpTo(String[] entities, int maxEvents) {
-        return null;
+    public static DataFindRequest oldestNewNotificationsForEntitiesUpTo(String[] entities,
```

Comment:
Actually, maybe just removing "New" would help. Would also make it consistent with terminology in `LightblueNotificationRepository`